python/paddle/fluid/imperative/layers.py

```diff
@@ -53,6 +70,44 @@ def full_name(self):
         """
         return self._full_name
 
+    @property
+    def param_attr(self):
+        return self._helper.param_attr
+
+    @property
+    def bias_attr(self):
+        return self._helper.bias_attr
+
+    def create_parameter(self,
+                         attr,
+                         shape,
+                         dtype,
+                         is_bias=False,
+                         default_initializer=None):
+        return self._helper.create_parameter(attr, shape, dtype, is_bias,
+                                             default_initializer)
+
+    def get_parameter(self, name):
+        return self._helper.get_parameter(name)
+
+    def create_variable_for_type_inference(self, dtype, stop_gradient=False):
+        return self._helper.create_variable_for_type_inference(dtype,
+                                                               stop_gradient)
+
+    def create_variable(self, *args, **kwargs):
+        return self._helper.main_program.current_block().create_var(*args,
+                                                                    **kwargs)
+
+    def append_bias_op(self, input_var, dim_start=1, dim_end=None):
```

Comment:
is append op enough?

Reply:
this func is used widely in old nn.py, we may need this func when we move those layers into imperative Layers